packages/amplify_datastore/example/lib/main.dart

```diff
@@ -116,7 +115,7 @@ class _MyAppState extends State<MyApp> {
     });
 
     (await Amplify.DataStore.query(Post.classType,
-            where: QueryField(fieldName: "post.id")
+            where: QueryField(fieldName: Post.ID.fieldName)
```

Comment:
you can just replace the whole thing with Post.ID which is a `QueryField` itself.

Reply:
Hmm this appears to be outdated, I've already done that update :            
 where: QueryField(fieldName: Post.ID.fieldName)

Reply:
but not `where: Post.ID..ne("e25859fc-e254-4e8b-8cae-62ccacce4097")`

Reply:
this is code for that: 

    (await Amplify.DataStore.query(Post.classType,
            where: QueryField(fieldName: Post.ID.fieldName)
                .ne("e25859fc-e254-4e8b-8cae-62ccacce4097")))
        .forEach((element) {
      postWithIdNotEquals += encoder.convert(element.toJson()) + '\n';
    });


It seems we're already using Post.ID.fieldName right?